browser/extensions/api/brave_rewards_api.cc

```diff
@@ -101,6 +101,26 @@ ExtensionFunction::ResponseAction BraveRewardsGetPublisherDataFunction::Run() {
   return RespondNow(NoArguments());
 }
 
+ExtensionFunction::ResponseAction
+  BraveRewardsGetTwitchPublisherDataFunction::Run() {
+  std::unique_ptr<brave_rewards::GetTwitchPublisherData::Params> params(
+    brave_rewards::GetTwitchPublisherData::Params::Create(*args_));
+  Profile* profile = Profile::FromBrowserContext(browser_context());
```

Comment:
do these functions need to check if profile is incognito or is that handled already?

Reply:
`incognito` is checked here: https://github.com/brave/brave-core/pull/888/files#diff-fc21efee53a5e1ba98eb186ef1e788a8R129